src/test/java/seedu/address/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,8 @@
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PERSON = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PERSON = Index.fromOneBased(3);
+
+    public static final Index INDEX_FIRST_GROUP = Index.fromOneBased(1);
+    public static final Index INDEX_SECOND_GROUP = Index.fromOneBased(2);
+    public static final Index INDEX_THIRD_GROUP = Index.fromOneBased(3);
```

Comment:
These indices also overlap with Ye Ji's one. In this case, it might cause merge conflicts. You may need to check.